package/ci/circleci.yml

```diff
@@ -224,14 +224,17 @@ commands:
           ninja install
 
   install-basis:
+    parameters:
+      tag:
+        type: string
+        default: "v1_50_0_2"
```

Comment:
I'd make the latest the default instead, and override it for older versions. So when there's a new version, one doesn't need to update several places.

Reply:
That's already latest, 1.50 😊

Reply:
Oh, my apologies. I saw it ending with 2 and being rather long, which made me think it's that old `v1_15_update2` tag. Their versioning is a mess.